docs/content/why-temporal.md

```diff
@@ -0,0 +1,168 @@
+---
+id: why-temporal
+title: What are the differences between Temporal and a traditional system?
+description: This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import CenteredImage from "../components/CenteredImage.js"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../content/what-is-a-workflow-execution.md'
+import * as WhatIsASignal from '../content/what-is-a-signal.md'
+import * as WhatIsAQuery from '../content/what-is-a-query.md'
+
+One of the aspects of the Temporal System is that it abstracts the complexity of a distributed system.
+Distributed systems exist to scale computation across multiple machines as the potential load of a system changes.
+In theory a distributed system facilitates a reliable and highly performant application.
+
+However there are a set of failures that can occur in a distributed application that can make things very complicated, especially at a large scale.
+
+<CenteredImage
+imagePath="/diagrams/basic-distributed-system.svg"
+imageSize="75"
+title="Distributed application failures"
+/>
+
+How will a downstream part of the application know if there was a failure before, failure during, failure between, or a failure after the changes to the state?
+How will the inconsistent state be reconciled if there was a failure between changes?
+
+In traditional systems, a large investment is often made to maintain the health of each individual component, visualize the health of the overall system, define timeout constraints for computations, orchestrate retries for computations that fail, and maintain a consistent state.
+
+These systems are often a mixture of stateless services, databases, cron jobs, and queues.
+And as these systems scale, responding to multiple asynchronous events, communicating with unreliable external resources, or tracking the state of something very complex becomes more and more challenging.
+
+Temporal reconfigures the use of services, databases, cron jobs, and queues into the Temporal Platform.
+The Temporal Platform addresses these failures head on and right out the box.
```

Comment:
We haven't discussed failures for three paragraphs, so the reference to "these failures" is ambiguous. Perhaps delete this sentence and extend the preceding sentence with "...and addresses failures head on."

(Side note: Our style guides inveigh against using "out [of] the box.")